de/refresher-linear-algebra.md

```diff
@@ -1,339 +1,339 @@
 
 <br>
 
 **31. Definitions**
 
-&#10230;
+&#10230; Definitionen
 
 <br>
 
 **32. Symmetric decomposition ― A given matrix A can be expressed in terms of its symmetric and antisymmetric parts as follows:**
 
-&#10230;
+&#10230; Symmetrische Zerlegung - Eine Matrix A kann durch dessen symmetrischen und schiefsymmetrischen Anteil wie folgt definiert werden:
 
 <br>
 
 **33. [Symmetric, Antisymmetric]**
 
-&#10230;
+&#10230; [Symmetrisch, Schiefsymmetrisch]
 
 <br>
 
 **34. Norm ― A norm is a function N:V⟶[0,+∞[ where V is a vector space, and such that for all x,y∈V, we have:**
 
-&#10230;
+&#10230; Matrixnorm - Die Matrixnorm is eine Funktion N:V⟶[0,+∞[ wobei V der Vektorraum ist und für alle x,y∈V gilt:
 
 <br>
 
 **35. N(ax)=|a|N(x) for a scalar**
 
-&#10230;
+&#10230; N(ax)=|a|N(x) wobei a ein Skalar ist
 
 <br>
 
 **36. if N(x)=0, then x=0**
 
-&#10230;
+&#10230; falls N(x)=0, folgt x=0**
 
 <br>
 
 **37. For x∈V, the most commonly used norms are summed up in the table below:**
 
-&#10230;
+&#10230; Sei x∈V, die häufigsten Matrixnormen werden in der folgenden Tabelle erläutert:
 
 <br>
 
 **38. [Norm, Notation, Definition, Use case]**
 
-&#10230;
+&#10230; [Matrixnorm, Notation, Definition, Verwendung]
 
 <br>
 
 **39. Linearly dependence ― A set of vectors is said to be linearly dependent if one of the vectors in the set can be defined as a linear combination of the others.**
 
-&#10230;
+&#10230; Lineare Abhängigkeit - Vektoren eines Vektorraums sind linear abhängig falls sich ein Vektor als Linearkombination der anderen Vektoren darstellen lässt.
 
 <br>
 
 **40. Remark: if no vector can be written this way, then the vectors are said to be linearly independent**
 
-&#10230;
+&#10230; Wichtig: falls kein Vektor als Linearkombination der anderen gebildet werden kann, werden die Vektoren als linear unabhängig bezeichnet.
 
 <br>
 
 **41. Matrix rank ― The rank of a given matrix A is noted rank(A) and is the dimension of the vector space generated by its columns. This is equivalent to the maximum number of linearly independent columns of A.**
 
-&#10230;
+&#10230; Rang - Der Rang einer Matrix A wird auch als rank(A) bezeichnet und wird durch die Dimension des Vektorraums, aufgespannt durch die Spaltenvektoren, definiert. Der Rang ist äquivalent mit der maximalen Anzahl an linear unabhängigen Spaltenvektoren von A.
 
 <br>
 
 **42. Positive semi-definite matrix ― A matrix A∈Rn×n is positive semi-definite (PSD) and is noted A⪰0 if we have:**
 
-&#10230;
+&#10230; Positiv-Semidefinierte Matrix - Eine Matrix A∈Rn×n ist positiv semidefinit (PSD), A⪰0 geschrieben, falls folgendes gilt:
 
 <br>
 
 **43. Remark: similarly, a matrix A is said to be positive definite, and is noted A≻0, if it is a PSD matrix which satisfies for all non-zero vector x, xTAx>0.**
 
-&#10230;
+&#10230; Wichtig: Ebenfalls gilt, Eine Matrix A ist positiv definit, A≻0, falls diese eine PSD Matrix ist und es gilt für alle Einheiten eines Vektors x ungleich 0: x, xTAx>0.
```

Comment:
&#10230; Wichtig: Ebenfalls gilt, dass eine Matrix A positiv definit ist, in der Notatition: A≻0, falls diese eine PSD Matrix ist und es gilt für alle Einheiten eines Vektors x ungleich 0: x, xTAx>0.